projects/m1/001-area-of-a-room/solutions/ranjit-t/index.js

```diff
@@ -0,0 +1,15 @@
+const length = parseInt(prompt("Enter Length of the room in meters"));
+const width = parseInt(prompt(" Super, Now Enter Width of the room in meters"));
+
+const area = length*width;
+
+
+
+if(isNaN(area))
+{
+  alert("make sure you enter right values");
+    
+}
+else {
+    alert(`The area of the room is ${area}`);
+}
```

Comment:
The code is not following the prettier  formatting rules. You should install prettier and set it up properly (turn on the "format on save option" in the vscode settings).

Reply:
Also please submit your solutions in a folder named ranjithtota (in the solutions folder)